tools/external_sort_test.py

```diff
@@ -0,0 +1,34 @@
+from csv_sort import external_sort
+import pandas as pd
+import os
+import unittest
+
+class TestExternalSortAccuracy(unittest.TestCase):
+    def test_accuracy(self):
```

Comment:
Think it would be good to add a comment underneath here to say exactly what this test case is testing.

Reply:
Maybe something like #Test that user id column is sorted correctly